### PR TITLE
add test for RI-4813 Do not duplicate Cloud databases

### PR DESCRIPTION
**Pull request by mariasergeenko**

### CODE REVIEW

File: tests/e2e/tests/critical-path/url-handling/url-handling.e2e.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ let { host, port, databaseName, databaseUsername = '', databasePassword = '' } =
 const redisConnect = 'redisinsight://databases/connect';
 
 fixture `Add DB from SM`
+    .only
```
Comment (vlad-dargel) [inline tests/e2e/tests/critical-path/url-handling/url-handling.e2e.ts:20]: .only

**Review by vlad-dargel — APPROVED** [review]
